.travis.yml

```diff
@@ -72,6 +72,12 @@ matrix:
         - java -version
         - mvn -version
         - mvn -B clean integration-test
+      cache:
+        directories:
+          - '$HOME/.m2/repository'
+          - '$HOME/.sonar/cache'
+          - '/c/java11'
+          - '/c/mvn361'
```

Comment:
only for windows?

Reply:
yes, only windows has /c/...
for other OSs, the setting is at the end of the file.